spring-cloud-aws-samples/spring-cloud-aws-dynamodb-sample/app/src/main/java/io/awspring/cloud/samples/dynamodb/SpringDynamoDbSample.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.samples.dynamodb;
+
+import io.awspring.cloud.dynamodb.DynamoDbOperations;
+import java.time.LocalDate;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.enhanced.dynamodb.Key;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+import software.amazon.awssdk.enhanced.dynamodb.model.PageIterable;
+import software.amazon.awssdk.enhanced.dynamodb.model.QueryConditional;
+import software.amazon.awssdk.enhanced.dynamodb.model.QueryEnhancedRequest;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+
+@SpringBootApplication
+public class SpringDynamoDbSample {
+
+	private DynamoDbOperations dynamoDbOperations;
+	private DynamoDbEnhancedClient dynamoDbEnhancedClient;
+	private static final Logger LOGGER = LoggerFactory.getLogger(SpringDynamoDbSample.class);
+
+	public SpringDynamoDbSample(DynamoDbOperations dynamoDbOperations, DynamoDbEnhancedClient dynamoDbEnhancedClient) {
+		this.dynamoDbOperations = dynamoDbOperations;
+		this.dynamoDbEnhancedClient = dynamoDbEnhancedClient;
+	}
+
+	public static void main(String[] args) {
+		SpringApplication.run(SpringDynamoDbSample.class, args);
+
+	}
+
+	@EventListener(ApplicationReadyEvent.class)
+	public void sendMessage() {
```

Comment:
lets turn it into `ApplicationRunner` bean and inject `DynamoDbOperations` and `DynamoDbEnhancedClient` through method parameters